packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-loading.tsx

```diff
@@ -0,0 +1,67 @@
+import { h } from '@stencil/core';
+
+export const DialLoading = () => (
+  <svg viewBox="0 0 276 276" data-testid="loading">
```

Comment:
do we already have a spinner component in the library?

Reply:
The sc-loading-spinner component just includes this dynamic circle. Then this component includes three parts with the outer ring , this dynamic circle and text 'loading'.